Server/Plugins/APIDump/Classes/BlockEntities.lua

```diff
@@ -1401,13 +1449,79 @@ World:ForEachChestInChunk(Player:GetChunkX(), Player:GetChunkZ(),
 				},
 				Notes = "Sets the spawn delay.",
 			},
+			SetSpawnCount =
+			{
+				Params =
+				{
+					{
+						Name = "SpawnCount",
+						Type = "number",
+					},
+				},
+				Notes = "Sets the number of entities the spawner will try to spawn in each activation. Might not spawn all of them due to spawn limitations of the entity.",
+			},
+			SetSpawnRange =
+			{
+				Params =
+				{
+					{
+						Name = "SpawnRange",
+						Type = "number",
+					},
+				},
+				Notes = "Sets the diameter of the square the spawner will try to spawn entities in.",
+			},
+			SetMinSpawnDelay =
+			{
+				Params =
+				{
+					{
+						Name = "MinSpawnDelay",
+						Type = "number",
+					},
+				},
+				Notes = "Sets the minimum amount of ticks the spawner will way before spawning new entities.",
+			},
+			SetMaxSpawnDelay =
+			{
+				Params =
+				{
+					{
+						Name = "MaxSpawnDelay",
+						Type = "number",
+					},
+				},
+				Notes = "Sets the maximum amount of ticks the spawner will way before spawning new entities.",
+			},
+			SetMaxNearbyEntities =
+			{
+				Params =
+				{
+					{
+						Name = "MaxNearbyEntities",
+						Type = "number",
+					},
+				},
+				Notes = "Sets the maximum amount of nearby entities until the spawner will stop spawning this entity type.",
+			},
+			SetRequiredPlayerRange =
+			{
+				Params =
+				{
+					{
+						Name = "RequiredPlayerRange",
+						Type = "number",
+					},
+				},
+				Notes = "Sets the maximum amount of blocks the spawner looks for a player during activation check.",
```

Comment:
This comment needs rewording,
```suggestion
				Notes = "Sets the maximum Euclidian distance from a player where the spawner can be activated.",
```

Reply:
It's not a euclidean distance though, that would describe a sphere not a box.

Reply:
Well that's what the implementation is doing, so if it should be Manhattan distance, then the implementation needs to be changed.

Reply:
Ah, you're right our implementation doesn't match the wiki. It should be a cube with side-length `2*RequiredPlayerRange`.

Reply:
The spawner actually looks for players in a sphere and spawns entities in a box

Reply:
Yep, according to the wiki our current behaviour of looking for players in a sphere is correct.